specification/cognitiveservices/data-plane/Language/stable/2022-05-01/conversationalanalysisauthoring.json

```diff
@@ -0,0 +1,3060 @@
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The project deletion job result.",
+            "schema": {
+              "$ref": "#/definitions/ConversationalAnalysisAuthoringProjectDeletionJobState"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Get Project Deletion Status": {
+            "$ref": "./examples/conversationalanalysisauthoring/SuccessfulGetProjectDeletionStatus.json"
+          }
+        }
+      }
+    },
+    "/authoring/analyze-conversations/projects/global/languages": {
+      "get": {
+        "description": "Lists the supported languages for the given project type.",
+        "operationId": "ConversationalAnalysisAuthoring_GetSupportedLanguages",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ConversationalAnalysisAuthoringProjectKindInQuery"
+          },
+          {
+            "$ref": "common.json#/parameters/TopParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/SkipParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/MaxPageSizeParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The list of supported languages.",
+            "schema": {
+              "$ref": "#/definitions/ConversationalAnalysisAuthoringSupportedLanguages"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Supported Languages": {
+            "$ref": "./examples/conversationalanalysisauthoring/SuccessfulGetSupportedLanguages.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "value"
+        }
+      }
+    },
+    "/authoring/analyze-conversations/projects/global/prebuilt-entities": {
+      "get": {
+        "description": "Lists the supported prebuilt entities that can be used while creating composed entities.",
+        "operationId": "ConversationalAnalysisAuthoring_GetSupportedPrebuiltEntities",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "language",
+            "description": "The language to get supported prebuilt entities for. Required if multilingual is false.",
```

Comment:
Same here. Wherever you mention the language, it's good to specify in what form it should be e.g., BCP-47.